compiler/rustc_lexer/src/unescape.rs

```diff
@@ -304,10 +298,10 @@ where
             '\t' => Ok('\t'),
             '"' => Err(EscapeError::EscapeOnlyChar),
             '\r' => Err(EscapeError::BareCarriageReturn),
-            _ => ascii_check(first_char, mode),
+            _ => ascii_check(c, mode),
         };
         let end = initial_len - chars.as_str().len();
-        callback(start..end, unescaped_char);
+        callback(start..end, result);
```

Comment:
Some functions in this crate use `result`, and some `res`. Better to unify to something. I guess we don't have any document where we document such trivial style decisions for compiler? (we do in rust-analyzer https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/style.md#variable-naming)

Reply:
I'll go with `res`.